app/api/bulk-operations/route.ts

```
@@ -0,0 +1,177 @@
+
+    // Start the database tracking
+    await supabase.rpc('start_bulk_operation', {
+      p_user_id: user.id,
+      p_operation_type: type,
+      p_record_count: recordIds.length
+    })
+
+    let result
+    switch (type) {
+      case 'tag_assign':
+        const tagAssignData = validatedData as z.infer<typeof BulkTagOperationSchema>
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)